### PR TITLE
'View in English' Button Link Fixed [Solves #6318]

### PR DESCRIPTION
**Pull request by HursitTarcan**
## Description

- Fixed the bug in the `View In English Button` (href).
- Changed `originalPagePath` to `/en/developers/`.  

## Related Issue
- Related to [#6318].

### CODE REVIEW

**Comment by HursitTarcan** [comment]
> Hey @HursitTarcan, thanks for taking a swing at this 💪
> 
> Hardcoding the English path doesn't quite solve the issue here.
> 
> The translation banner should have a 'View in English' button that routes the user to the exact same page they are on but in English. This banner can appear on almost any page (here is another example: https://ethereum.org/ro/staking/) so it needs to be generated programmatically.

Ohw ok I understand, I thought it was only on this specific page. I will try to solve this.

File: src/components/TranslationBanner.js [hunk 1]
```diff
@@ -144,7 +144,7 @@ const TranslationBanner = ({
             </div>
             {!isPageContentEnglish && (
               <div>
-                <SecondaryButtonLink isSecondary to={`${originalPagePath}`}>
+                <SecondaryButtonLink isSecondary to={`${"/en" + originalPagePath}`}>
```
Comment (corwintines) [inline src/components/TranslationBanner.js:147]:
```suggestion
                <SecondaryButtonLink isSecondary to={`/en${ originalPagePath}`}>
```
Just small suggestion to clean up code

**Comment by corwintines** [comment]
@HursitTarcan left a small change request to clean up code, but this looks good other than that!

**Comment by HursitTarcan** [comment]
> @HursitTarcan left a small change request to clean up code, but this looks good other than that!

Changed it! Thanks for the help <3

**Review by corwintines — APPROVED** [review]
LGTM! Thanks @HursitTarcan :)

**Pull request merged**